lib/wings/orm_converter.rb

```diff
@@ -48,15 +40,9 @@ def self.base_for(klass:)
     #   results in long methods
     def self.to_valkyrie_resource_class(klass:)
       relationship_keys = klass.respond_to?(:reflections) ? relationship_keys_for(reflections: klass.reflections) : []
-      relationship_keys.delete('member_ids')
-      relationship_keys.delete('member_of_collection_ids')
       reflection_id_keys = klass.respond_to?(:reflections) ? klass.reflections.keys.select { |k| k.to_s.end_with? '_id' } : []
 
       Class.new(base_for(klass: klass)) do
-        include Wings::CollectionBehavior if klass.included_modules.include?(Hyrax::CollectionBehavior)
```

Comment:
this is the biggest change here. no longer injecting Hydra::Works-like behavior into valkyrie resources at runtime. now relying on the `Hyrax::FileSet`, `Hyrax::PcdmCollection` and `Hyrax::Work` classes to provide all the model layer behavior we want.